docs/parallel_training/features.rst

```diff
@@ -0,0 +1,53 @@
+Features
+========
+
+Highlights
+##########
+
+Simple usage
+^^^^^^^^^^^^
+| With a single `@parl.remote_class` decorator, we can access cluster's remote
+ computation resources to run our jobs easily. We can add any machine at
+ anytime to the cluster, and PARL will take care of the network communication
+ or resources allocation for us.
+
+High performance
+^^^^^^^^^^^^^^^^
+| `@parl.remote_class` enable us to achieve real multi-thread computation
+ efficiency without modifying our codes. As shown in figure (a), python's
+ original multi-thread computation performs poorly due to the limitation
+ of the GIL, while PARL empowers us to realize real parallel computation
+ efficiency.
+
+Convenient visualization
```

Comment:
Computation Resources Monitor.
PARL provides a web monitor to watch the status of any resources connected to the cluster.